hack/revert-ci-operator.sh

```diff
@@ -3,6 +3,6 @@
 set -euo pipefail
 
 
-OLD_LATEST="$(oc --context api.ci get is ci-operator -n ci -o jsonpath={.status.tags[?\(@.tag==\"latest\"\)].items[1].dockerImageReference}|cut -d '@' -f2)"
+OLD_LATEST="$(oc --context app.ci get is ci-operator -n ci -o jsonpath={.status.tags[?\(@.tag==\"latest\"\)].items[1].dockerImageReference}|cut -d '@' -f2)"
```

Comment:
Added by mistake?

Reply:
Yeah, moved it to https://github.com/openshift/ci-tools/pull/1849